File: 0334-increasing-triplet-subsequence/0334-increasing-triplet-subsequence.go
func increasingTriplet(nums []int) bool {
    low := 1 << 31;
    mid := 1 << 31;
    
    for _, n := range nums {
        if n <= low {
            low = n
        } else if n <= mid {
            mid = n
        } else {
            return true
        }
    }
    
    return false
}

File: 0334-increasing-triplet-subsequence/0334-increasing-triplet-subsequence.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var increasingTriplet = function(nums) {
    let low = Infinity;
    let mid = Infinity;
    
    for (const n of nums) {
        if (n <= low) {
            low = n;
        } else if (n <= mid) {
            mid = n;
        } else {
            return true;
        }
    }
    
    return false;
};